packages/client/src/components/SafeSettings/components/SafeDetailsSetting/EditSafeName.js

```diff
@@ -0,0 +1,50 @@
+import { useState } from "react";
+
+const EditSafeName = ({ name, onCancel, onSubmit }) => {
+  const [currSafeName, setCurrSafeName] = useState(name);
```

Comment:
nit: we can rename to safeName

Reply:
I actually think we should keep the edit part, just to differentiate it from other components that only displays safe name, like the one on home screen

Reply:
currSafeName -> safeName

Reply:
lol I see, I thought you were talking about the file name, my bad, yes good idea!